src/main/java/mate/academy/bookstore/controller/AuthenticationController.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.bookstore.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.user.UserRegistrationRequestDto;
+import mate.academy.bookstore.dto.user.UserResponseDto;
+import mate.academy.bookstore.exception.RegistrationException;
+import mate.academy.bookstore.service.RegistrationService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "User manager", description = "Endpoints for user authentication")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/auth")
```

Comment:
```suggestion
@RequestMapping("/auth")
```

use `server.servlet.context-path=/api` in application.properties